src/renderer/kube-object.store.ts

```diff
@@ -198,3 +198,11 @@ export abstract class KubeObjectStore<T extends KubeObject = any> extends ItemSt
     this.items.replace(this.sortItems(items));
   }
 }
+
+export interface Statuses {
+  [key: string]: number;
+}
+
+export abstract class StatusKubeObjectStore<T extends KubeObject = any> extends KubeObjectStore<T> {
```

Comment:
Don't really like that it creates extra class inheritance only because of this.
Let's add optional method instead to base class:

```
  // Optional methods
  getStatuses?(items: T[]): Record<string, number>;
```